AirportDictionary/AirportDictionary/Pages/Index.cshtml.cs

```diff
@@ -20,47 +20,42 @@ public IndexModel(ILogger<IndexModel> logger)
             _logger = logger;
         }
 
-        public void OnGet()
+        public JsonResult OnGet()
         {
 
-            using (var webClient = new System.Net.WebClient())
-            {
-
-                //IDictionary<long, QuickTypeCity.Welcome> allCities = new Dictionary<long, QuickTypeCity.Welcome>();
-
-                string cityJson = webClient.DownloadString("https://pkgstore.datahub.io/core/world-cities/world-cities_json/data/5b3dd46ad10990bca47b04b4739a02ba/world-cities_json.json");
+                string cityJson = getJsonData("https://pkgstore.datahub.io/core/world-cities/world-cities_json/data/5b3dd46ad10990bca47b04b4739a02ba/world-cities_json.json");
                 QuickTypeCity.Welcome[] allCity = QuickTypeCity.Welcome.FromJson(cityJson);
 
-                ViewData["Welcome"] = allCity;
-
-                //foreach (QuickTypeCity.Welcome welcome in allCity)
```

Comment:
Good to remove commented-out code.  It's considered a no-no to commit commented-out code to version control, because you can always go back in history to see what used to be there.